types/store/rhs.ts

```diff
@@ -24,6 +24,8 @@ export type PostDraft = {
     uploadsInProgress: string[];
     props?: any;
     caretPosition?: number;
+    channel_id: string;
```

Comment:
Could you rename that `channelId` so that we're not mixing snake case and camel case here?